float64 leg_ang_frq
float64 leg_ang_amp
float64 leg_len_frq
float64 leg_len_amp
float64 hip_amp
float64 hip_frq
float64 p_gain
float64 d_gain
float64 hip_p_gain
float64 hip_d_gain
